integration_test/agents/agents.go

```diff
@@ -188,13 +191,26 @@ func RunOpsAgentDiagnostics(ctx context.Context, logger *logging.DirectoryLogger
 		"/etc/google-cloud-ops-agent/config.yaml",
 		"/var/log/google-cloud-ops-agent/subagents/logging-module.log",
 		"/var/log/google-cloud-ops-agent/subagents/metrics-module.log",
-		"/run/google-cloud-ops-agent-fluent-bit/fluent_bit_main.conf",
-		"/run/google-cloud-ops-agent-fluent-bit/fluent_bit_parser.conf",
 		"/run/google-cloud-ops-agent-opentelemetry-collector/otel.yaml",
 	} {
 		_, basename := path.Split(log)
 		gce.RunRemotely(ctx, logger.ToFile(basename+txtSuffix), vm, "", "sudo cat "+log)
 	}
+
+	// Capture all files beneath /run/google-cloud-ops-agent-fluent-bit
+	fluentBitDir := "/run/google-cloud-ops-agent-fluent-bit"
```

Comment:
Maybe just grab /run/google-cloud-ops-agent* in its entirety and keep the existing paths? That seems simpler and more consistent.